### PR TITLE
Moves to Jitpack for service manager and cleans logs

### PR DESCRIPTION
**Pull request by adriancole**
This makes logging more succinct and clear, especially when grepping for 'since'

```bash
pinot                   | 08:46:22,986 INFO  [main] command.StartServiceManagerCommand (StartServiceManagerCommand.java:285) - Starting a Pinot [SERVICE_MANAGER] at 0.007s since launch
pinot                   | 08:46:27,220 INFO  [main] command.StartServiceManagerCommand (StartServiceManagerCommand.java:287) - Started Pinot [SERVICE_MANAGER] instance [ServiceManager_20da8f16c3fb_7098] at 4.242s since launch
pinot                   | 08:46:27,246 INFO  [main] command.StartServiceManagerCommand (StartServiceManagerCommand.java:285) - Starting a Pinot [CONTROLLER] at 4.268s since launch
pinot                   | 08:46:32,430 INFO  [main] command.StartServiceManagerCommand (StartServiceManagerCommand.java:287) - Started Pinot [CONTROLLER] instance [Controller_20da8f16c3fb_9000] at 9.452s since launch
pinot                   | 08:46:32,432 INFO  [Start a Pinot [BROKER]] command.StartServiceManagerCommand (StartServiceManagerCommand.java:285) - Starting a Pinot [BROKER] at 9.453s since launch
pinot                   | 08:46:32,432 INFO  [Start a Pinot [SERVER]] command.StartServiceManagerCommand (StartServiceManagerCommand.java:285) - Starting a Pinot [SERVER] at 9.453s since launch
pinot                   | 08:46:35,755 INFO  [Start a Pinot [BROKER]] command.StartServiceManagerCommand (StartServiceManagerCommand.java:287) - Started Pinot [BROKER] instance [Broker_20da8f16c3fb_8099] at 12.777s since launch
pinot                   | 08:46:35,972 INFO  [Start a Pinot [SERVER]] command.StartServiceManagerCommand (StartServiceManagerCommand.java:287) - Started Pinot [SERVER] instance [Server_20da8f16c3fb_8098] at 12.993s since launch
pinot                   | *** Starting schema installation at 15s since container launch
pinot                   | *** Completed schema installation at 17s since container launch
```

### CODE REVIEW

File: pinot-servicemanager/Dockerfile [hunk 1]
```diff
@@ -1,22 +1,16 @@
...
 
-WORKDIR /install
+# Using the same image as we use in CircleCI to avoid transfer costs
+FROM cimg/openjdk:14.0.2 AS install
+
+# Override to build an image from a fork. Ex. kotharironak
+ARG JITPACK_USER=apache
+# Override to use a different expression, such as 'master-SNAPSHOT'
+#   Note: 0.5.0 does not have parallel start (PR 5917): 21a372b2f58f9c6e27fe9710d6952ed519342061
+ARG JITPACK_TAG=23797914c6fea24e34f3ebfca9a73e0fff72c2b7
```
Comment (adriancole) [inline pinot-servicemanager/Dockerfile:10]: jitpack used to not work, but it is ok now.

File: pinot-servicemanager/docker-bin/install-schema [hunk 2]
```diff
@@ -17,24 +17,24 @@ http_post() {
   echo
 }
 
+elapsed_seconds() {
```
Comment (adriancole) [inline pinot-servicemanager/docker-bin/install-schema:20]: this is super handy!

File: pinot-servicemanager/docker-bin/install-schema [hunk 3]
```diff
@@ -17,24 +17,24 @@ http_post() {
...
 timeout=180
-echo "Will wait up to ${timeout} seconds for Pinot Server to come up before installing Schema"
-while [[ "$timeout" -gt 0 ]] && ! wget -qO- http://${IP}:8097/health > /dev/null 2>&1; do
+echo "Will wait up to ${timeout} seconds for Pinot Broker to register with controller"
+while [[ "$timeout" -gt 0 ]] && ! (wget -qO- http://${IP}:9000/brokers/tenants 2>&-| grep -q DefaultTenant); do
     sleep 1
     timeout=$(($timeout - 1))
 done
 
-# Wait for broker to registers
```
Comment (adriancole) [inline pinot-servicemanager/docker-bin/install-schema:29]: no need for double wait loops

File: pinot-servicemanager/install.sh [hunk 4]
```diff
@@ -4,6 +4,20 @@
...
+  # Download scripts and config for Kafka and ZooKeeper, but not for Connect
+  wget -qO temp.zip https://jitpack.io/com/github/${JITPACK_USER}/incubator-pinot/${artifactId}/${JITPACK_TAG}/${artifactId}-${JITPACK_TAG}-shaded.jar
+  # Pinot starts faster when classes are extracted
+  unzip -qo temp.zip -d classes
+  # remove license because sometimes a file and other times a directory
+  rm -rf temp.zip classes/META-INF/license
+done
+
+# TODO: try maven-dependency-plugin:unpack instead of wget
+#       https://maven.apache.org/plugins/maven-dependency-plugin/examples/unpacking-artifacts.html
```
Comment (adriancole) [inline pinot-servicemanager/install.sh:19]:
```suggestion
#       https://maven.apache.org/plugins/maven-dependency-plugin/examples/unpacking-artifacts.html
#       https://github.com/hypertrace/pinot/issues/16
```

**Comment by adriancole** [comment]
adding more specific logs to CI so we aren't accidentally counting time spent pulling kafka

**Comment by jcchavezs** [comment]
LGTM

**Comment by adriancole** [comment]
GitHub Action:
```
sut                | *** Completed schema installation at 33s since container launch
```

CircleCI
```
*** Completed schema installation at 52s since container launch
```

**Comment by adriancole** [comment]
ok I think this will help us debug. I noticed on my laptop, if I only allocate 2GB to docker, startup takes almost a minute sometimes longer. If I allocate 6GB it is quite fast.

Merging so we can use similar debugging in hypertrace/hypertrace

**Pull request merged**